parm/config/gefs/config.wave

```diff
@@ -121,15 +118,15 @@ export RSTIOFF_WAV=0                   # first restart file offset relative to m
 #
 # Set runmember to default value if not GEFS cpl run
 #  (for a GFS coupled run, RUNMEN would be unset, this should default to -1)
-export RUNMEM=${RUNMEM:--1}
+export RUNMEM="-1"
 # Set wave model member tags if ensemble run
 # -1: no suffix, deterministic; xxxNN: extract two last digits to make ofilename prefix=gwesNN
-if [[ ${RUNMEM} = -1 ]]; then
+if (( RUNMEM == -1 )); then
 # No suffix added to model ID in case of deterministic run
-  export waveMEMB=
+  export waveMEMB=""
 else
 # Extract member number only
-  export waveMEMB="${RUNMEM: -2}"
+  export waveMEMB="${RUNMEM}"
 fi
```

Comment:
Do we need this section at all then, if we are just making it `-1`?

Reply:
I couldn't find where `RUNMEM` is set before line 124. 
If `RUNMEM` will always be `-1` for `gefs` then I agree this can be consolidated.

Reply:
@JessicaMeixner-NOAA may remember why this was in the first place.

Reply:
I'm assuming this was here as the original attempt to unify GEFSv12 and GFSv16 scripts for waves.  They were separated fairly early on in the process but hopefully will get unified as much as possible with GEFSv13 and GFSv17.